### PR TITLE
Wireframe

### PR DESCRIPTION
**Pull request by ysoseriouz**
Step 1: draw wireframe fulfill user story

### CODE REVIEW

**Review by ducpt1012 — COMMENTED** [review]
- These wireframe files should be moved into a folder /wireframes.
- You should remove these unnecessary before add and commit.  Example:   .DS_Store ...
image

**Comment by ysoseriouz** [comment]
Okk I get it

**Comment by ysoseriouz** [comment]
Should I use gitignore for this kind of files? I'd love to understand how .gitignore works

File: requirement.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- ...
+
+## User story
+0. User sign-in, sign-up (username & password) -> go to main page with navigation bar (option: 'Inventory', 'Trash').
+1. User at main page 'inventory_list': viewing a list of phones in storage.
+    - At the top of the page, there's a search/filter box, a combination of queried properties is used to match inventories -> only view matched phones.
+    - User clicks button 'Add'/'Add inventory' to register new inventory -> no. 2
+    - User clicks icon 'Edit' -> go to 'edit_inventory' -> no. 3
+    - User clicks icon 'Delete' -> the row/phone will be delete from inventory list (but keep it on DB maybe later need..)
+    - User clicks 'Trash' in navigation bar -> go to 'trash_list' -> no.4
```
Comment (dptvn) [inline requirement.md]: IMO, it could be combined with `inventory_list` screen. 
You just add a check box `with_delete` in  inventory_list screen

**Review by ducpt1012 — COMMENTED** [review]
1. Do we need these wireframes for `brand`?
2. Search form: 
 - Year: drop list
 - Price: should display price in the search form, and display origin_price on the result
 - Percent: should use meaning name, such as status: new, used > 90%, used >80%  ...
 - Condition sort? : IMO, it better than to use header sort in the result table.
 - Reset button: is necessary?

**Comment by ysoseriouz** [comment]
2. 
- Ok drop list for 'year' is reasonable
- I forgot about the result, I'll fix
- About the dropbox (maybe I will note the details on requirement.md as you suggested, it's may not clear on my wireframes)
- The sorting one at header of table is reasonable
- I think of a situation that one would have filled some fields but want to search different combination -> just a button to delete typed filled, I will think about the UI of this.

1. I don't get the question..

File: requirement.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- Memory size (String): ie. 256 GB
+- Manufacturing year (Number)
+- OS version (String)
+- Color (String)
+- Price (Number) (price for sale - VND)
+
+Optional:
+- Original Price (price when phone is bought from the owner)
+- Source (phone used to belong to whom..)
+- Percent (how many percent new of the used-phone)
```
Comment (dptvn) [inline requirement.md]: Don't forget `percent` here

**Review by ducpt1012 — APPROVED** [review]
LGTM 🦖

**Pull request merged**